### DISCUSSION TITLE
plotting stopping at 11%

### DISCUSSION THREAD

**Discussion by IronM6n** [Chia-Network/chia-blockchain#8231] [Support]
no errors  during plotting>  i have intel NUC with 8gigs of ram.  saving the plot on external drive 
![Capture](https://user-images.githubusercontent.com/84716917/131065912-a5d24a81-c723-4638-8977-8c1c0b2efd52.PNG)

**Comment by loppefaaret** [comment]
Often, this is an indication of hardware issues - could you perhaps try and go into your BIOS, and try and disable "XMP Profiles" - and make sure your machine aren't over heating ?

**Reply by IronM6n** [reply]
thanks, im trying few more things

**Reply by IronM6n** [reply]
i think i found the root problem. i ran repair on drive so far its 14%  which is good sign.  thanks for your responses

**Comment by BrandtH22** [comment]
@IronM6n , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
